projects/olympics/index.js

```diff
@@ -7,23 +7,21 @@ import { useTracking } from "analytics/context";
 import { BarChart } from "./BarChart";
 import PoleVis from "./PoleVis";
 import CustomSelect from "@components/select/select";
-
+import { getOlympicDataToUse } from "./utils";
```

Comment:
This still needs to load all the data, we should just trim the file to contain the relevant data (ie all of the countries who were at the top at some point, but we don't need the data for Chile for example)

Reply:
🤔  You mean like actually remove the countries that never reached the top 11? 

And yes, it load all the data but does a single slice to every year once the component mounted and not on every render. 

at the end the data that loads it's max 11 country for every year.
![image](https://user-images.githubusercontent.com/12849040/129409515-fdcca8cb-69e3-4320-9817-1280c4f4766e.png)

this is what you meant? @sikolio